vnet/lib/vnet/models/interface.rb

```diff
@@ -39,6 +40,7 @@ class Interface < Base
     active_interfaces: :destroy,
     datapath_networks: :destroy,
     datapath_route_links: :destroy,
+    filters: :destroy,
```

Comment:
Filters is not part of 0001_origin, so put it at the bottom with the appropriate migration filename.